diff -ur sqlmap.orig/lib/core/common.py sqlmap/lib/core/common.py
--- sqlmap.orig/lib/core/common.py	2011-04-10 15:29:21.000000000 +0200
+++ sqlmap/lib/core/common.py	2011-04-26 20:55:38.309567115 +0200
@@ -846,7 +846,7 @@
     paths.SQLMAP_UDF_PATH = os.path.join(paths.SQLMAP_ROOT_PATH, "udf")
     paths.SQLMAP_XML_PATH = os.path.join(paths.SQLMAP_ROOT_PATH, "xml")
     paths.SQLMAP_XML_BANNER_PATH = os.path.join(paths.SQLMAP_XML_PATH, "banner")
-    paths.SQLMAP_OUTPUT_PATH = os.path.join(paths.SQLMAP_ROOT_PATH, "output")
+    paths.SQLMAP_OUTPUT_PATH = os.path.expanduser('~/.sqlmap')
     paths.SQLMAP_DUMP_PATH = os.path.join(paths.SQLMAP_OUTPUT_PATH, "%s", "dump")
     paths.SQLMAP_FILES_PATH = os.path.join(paths.SQLMAP_OUTPUT_PATH, "%s", "files")
 
diff -ur sqlmap.orig/sqlmap.py sqlmap/sqlmap.py
--- sqlmap.orig/sqlmap.py	2011-04-08 12:56:38.000000000 +0200
+++ sqlmap/sqlmap.py	2011-04-26 20:50:51.263102512 +0200
@@ -23,6 +23,8 @@
 except ImportError:
     pass
 
+sys.path.insert(0, '/usr/share/sqlmap')
+
 from lib.controller.controller import start
 from lib.core.common import banner
 from lib.core.common import dataToStdout
@@ -54,7 +56,7 @@
     if weAreFrozen():
         return os.path.dirname(getUnicode(sys.executable, sys.getfilesystemencoding()))
     else:
-        return os.path.dirname(os.path.realpath(__file__))
+	return '/usr/share/sqlmap'
 
 def main():
     """
