--- sqlmap-0.6.4/sqlmap.py~	2009-04-15 14:34:45.592581739 +0300
+++ sqlmap-0.6.4/sqlmap.py	2009-04-15 14:47:33.299271416 +0300
@@ -36,6 +36,8 @@
 except ImportError, _:
     pass
 
+sys.path.insert(0, '/usr/share/sqlmap')
+
 from lib.controller.controller import start
 from lib.core.common import banner
 from lib.core.common import setPaths
@@ -58,7 +60,7 @@
     if weAreFrozen():
         return os.path.dirname(unicode(sys.executable, sys.getfilesystemencoding()))
     else:
-        return os.path.dirname(os.path.realpath(__file__))
+        return '/usr/share/sqlmap'
 
 
 def main():
--- sqlmap-0.6.4/lib/core/common.py~	2009-01-28 16:53:11.000000000 +0200
+++ sqlmap-0.6.4/lib/core/common.py	2009-04-15 15:25:58.012270275 +0300
@@ -513,7 +513,7 @@
     paths.SQLMAP_TXT_PATH        = "%s/txt" % paths.SQLMAP_ROOT_PATH
     paths.SQLMAP_XML_PATH        = "%s/xml" % paths.SQLMAP_ROOT_PATH
     paths.SQLMAP_XML_BANNER_PATH = "%s/banner" % paths.SQLMAP_XML_PATH
-    paths.SQLMAP_OUTPUT_PATH     = "%s/output" % paths.SQLMAP_ROOT_PATH
+    paths.SQLMAP_OUTPUT_PATH     =  os.path.expanduser('~/.sqlmap')
     paths.SQLMAP_DUMP_PATH       = paths.SQLMAP_OUTPUT_PATH + "/%s/dump"
     paths.SQLMAP_FILES_PATH      = paths.SQLMAP_OUTPUT_PATH + "/%s/files"
 
